### DISCUSSION TITLE
KITT4SME Getting started

### DISCUSSION THREAD

**Discussion by pietrofusco** [c0c0n3/kitt4sme.live#225] [Q&A]
Dear Community,
I am Pietro and I am a new on KITT4SME platform. 

I would like to kindly ask if there is any tutorial/wizard which is able to introduce me on how to use/deploy an AI-based module developed by me, using Pytorch and Python, on KITT4SME.

I hope I made my point,

Best regards,

Pietro.

**Comment by kostasgrevenitis** [comment]
Hello. 

You can start from here. 

Read the "Docker compose based KITT4SME platform".

Leave the "Kubernetes based KITT4SME platform" for now

https://github.com/c0c0n3/kitt4sme.live/blob/open-calls/docs/bootstrap.md
